src/pages/ResourceForm.tsx

```diff
@@ -196,8 +197,18 @@ const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) =
                 <StyledPanel>
                   {isLoadingLicenses && <CircularProgress />}
                   {loadingLicensesErrorStatus !== StatusCode.ACCEPTED && <ErrorBanner />}
+                  <StyledSchemaPart>
+                    <StyledContentWrapper>
+                      <Typography variant="h4">{formikProps.values.resource.features.dlr_title}</Typography>
```

Comment:
Gjerne kopier med tittelen på de andre fanene også - ser vi (jeg) har glemt det.

Reply:
Fikset, har latt tittelen på fanen stå på resourceForm enn så lenge. På lang sikt kan vi sikkert ordne en fil per fane som holder orden alt inni fanen. Men akkurat nå syntes jeg det ble rotete hvis fila som har innholdet for AccessAndLicense skal inneholde tittelen, når tittelen blir liggende i resourceForm i de andre fanene.